sbt-version-policy/src/main/scala/sbtversionpolicy/SbtVersionPolicyPlugin.scala

```diff
@@ -17,6 +19,9 @@ object SbtVersionPolicyPlugin extends AutoPlugin {
     )
 
   object autoImport extends SbtVersionPolicyKeys {
+    // Note: the key is defined here rather than in the trait `SbtVersionPolicyKey` because adding a
+    // member to a trait would break binary compatibility.
+    final val versionPolicyIgnoredInternalDependencyVersions = settingKey[Option[Regex]]("Exclude dependencies to projects of the current build whose version matches this regular expression.")
```

Comment:
I don’t understand why I had to do this hack although sbt-version-policy uses Scala 2.12, which should support adding new methods to traits?

Reply:
It seems it’s not possible to add a `val` to a trait, but a `def` is OK. See https://github.com/lightbend/mima/blob/main/functional-tests/src/test/trait-added-val-in-new-version-nok/problems.txt